src/Domain/CoordinateMaps/Wedge3D.hpp

```diff
@@ -27,8 +27,6 @@ namespace CoordinateMaps {
  *  wedge centered on a coordinate axis covering a volume between an inner
  *  surface and outer surface. The inner surface can be given a curvature
  *  between flat (a sphericity of 0) or spherical (a sphericity of 1).
- *  The sphericity of the outer surface is currently constrained to have a
- *  value of 1.
  *
  *  The first two logical coordinates correspond to the two angular coordinates,
  *  and the third to the radial coordinate.
```

Comment:
There are references to the "spherical face" and the "bulged face" in the next paragraph.  (Outside the diff.)